truss/main.go

```diff
@@ -106,9 +117,9 @@ func parseInput() (*truss.Config, error) {
 		cfg.PBPackage = cfg.ServicePackage
 	} else {
 		cfg.PBPackage = *pbPackageFlag
-		if !fileExists(
-			filepath.Join(cfg.GOPATH, "src", cfg.PBPackage)) {
-			return nil, errors.Errorf(".pb.go output package directory does not exist: %q", cfg.PBPackage)
+		PBPackagePath := filepath.Join(cfg.GOPATH, "src", cfg.PBPackage)
+		if !fileExists(PBPackagePath) {
+			return nil, errors.Errorf(".pb.go output package directory does not exist: %q", PBPackagePath)
```

Comment:
I like the breaking this variable out so we can get error messages with the absolute path.